model-optimizer/extensions/front/kaldi/replace_dropoutmask.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+from extensions.middle.MakeKaldiConstReshapable import create_const_with_batch_from_input
+from mo.front.common.replacement import FrontReplacementPattern
+from mo.graph.graph import Graph
+
+
+class ReplaceDropoutMaskPattern(FrontReplacementPattern):
+    enabled = True
+    run_non_recursive = True
+
+    def run_after(self):
+        from extensions.front.restore_ports import RestorePorts
+        return [RestorePorts]
+
+    def run_before(self):
+        from extensions.front.kaldi.replace_lstm_nonlinearity import ReplaceLstmNonLinearityPattern
+        return [ReplaceLstmNonLinearityPattern]
+
+    def find_and_replace_pattern(self, graph: Graph):
+        batch_port = graph.get_op_nodes(op="Parameter")[0].out_port(0)
```

Comment:
can it be that a different operation node used instead of `Parameter` node?

Reply:
We have transformations, where another node used to take shape (input of node usually) but dropoutmask have no input, so Parameter is used.